mysql-test/suite/rocksdb/t/add_index_inplace.test

```diff
@@ -255,7 +255,7 @@ SET @prior_rocksdb_merge_buf_size = @@rocksdb_merge_buf_size;
 
 SET global rocksdb_strict_collation_check = off;
 SET session rocksdb_merge_combine_read_size = 566;
-SET session rocksdb_merge_buf_size = 336;
+SET session rocksdb_merge_buf_size = 340;
```

Comment:
To understand why this is happening we need to look at the full test case

```
SET global rocksdb_strict_collation_check = off;
SET session rocksdb_merge_combine_read_size = 566;
SET session rocksdb_merge_buf_size = 336;

show variables like '%rocksdb_bulk_load%';
CREATE TABLE t1 (a VARCHAR(80)) ENGINE=RocksDB;
INSERT INTO t1 (a) VALUES (REPEAT("a", 80));
INSERT INTO t1 (a) VALUES (REPEAT("a", 80));
INSERT INTO t1 (a) VALUES (REPEAT("a", 80));
INSERT INTO t1 (a) VALUES (REPEAT("a", 80));
ALTER TABLE t1 ADD INDEX ka(a), ALGORITHM=INPLACE;
SHOW CREATE TABLE t1;
CHECK TABLE t1;
--sorted_result
SELECT * FROM t1 FORCE INDEX(ka) WHERE a > "";
DROP TABLE t1;
```

`rocksdb_combine_read_size` is the total amount of space we have available to allocate for the heap that we use for the combine phase of our external merge sort during inplace alter.  

The relevant calculation is 
https://github.com/facebook/mysql-5.6/blob/fb-mysql-5.6.35/storage/rocksdb/rdb_index_merge.cc#L237-L241

The problem here is that there are now more sort buffers because of the bitmap stored in each record takes extra space. (Note that we should be able to avoid this. if the index is on the full key then its always covering, i don't think we should store the bitmap in these cases).

The end result is that the alter can't continue because there isn't enough space in the chunk to process the merge.  We should also return a better error to the user when this happens (currently it's only written to the error logs).  I'll open a task for it later.